app/views/sorns/search.html.erb

```diff
@@ -109,6 +109,23 @@
                     </div>
                   </div>
 
+                  <% if sorn.mentioned.present? %>
+                    <div class="grid-row">
+                      <div class="grid-col-12">
+                        <span class="sorn-attribute-header">Other SORNs referenced:</span>
+                        <ul>
+                        <% sorn.mentioned.each do |metioned_sorn| %>
+                          <li>
+                            <a href="/search?search=<%= metioned_sorn.citation %>&fields[]=citation"><%= metioned_sorn.citation %></a>:
+                            <%= metioned_sorn.action %>,
```

Comment:
potentially as a future enhancement, we could consider trying :action_type here now that we have it.